echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/PipelineCache.java

```diff
@@ -132,6 +148,40 @@ void pollPipelineConfigs() {
     }
   }
 
+  private List<Pipeline> fetchHydratedPipelines() {
+    List<Pipeline> rawPipelines = front50.getPipelines();
+    if (rawPipelines == null) {
+      return Collections.EMPTY_LIST;
+    }
+
+    Map<Boolean, List<Pipeline>> partitionedByTemplated = rawPipelines.stream()
```

Comment:
Agreed, it may make sense to just pull the hydration into a class of its own and tests to verify that a single orca call (or problem with a pipeline) will not impact the entire pipeline cache.

The existing catch block doesn't entirely include the new behavior, there's a `front50.errors` metric but we've now introduced some calls to orca.

Reply:
Simplified this bad boy and added metrics for the orca calls.